Vision/AutoMLVisionClassifier/powerskill/models.py

```diff
@@ -0,0 +1,83 @@
+import json
+import logging
+
+from azureml.contrib.automl.dnn.vision.common.model_export_utils import load_model
+from azureml.core import Workspace
+from azureml.core.authentication import ServicePrincipalAuthentication
+from azureml.core.experiment import Experiment
+
+
+class Models:
+
+    def __init__(self, azureml_model_dir, classication_model):
+        self.azureml_model_dir = azureml_model_dir
+        self.classication_model = classication_model
+        self.task_type = 'image-multi-labeling'  # TODO Change to project type when required
+
+    def load_classification_model(self, azureml_model_dir):
+        """
+        Load the model
+        :param azureml_model_dir: Check if locally mounted or download latest model
+        :return: Load AML model
+        """
+        logging.info(f"Loading model from {azureml_model_dir}")
+        model_settings = {}
+        self.classication_model = load_model(self.task_type, azureml_model_dir, **model_settings)
+        self.azureml_model_dir = azureml_model_dir
+
+    def get_workspace(self):
+        """
+        # Uses the config.json file to load the AML workspace
+        :return: An AML workspace instance
+        """
+
+        # Our AML config file
+        with open("/usr/src/api/config.json", "r") as json_file:
```

Comment:
Can we move that path to an environment variable with this value as default? Also, can we construct that path with a more x-plat-friendly method?

Reply:
This path is specified by the dockerfile during build and will be constant - the platform is also constant.